akka-pki/src/test/resources/README.md

```diff
@@ -0,0 +1,3 @@
+# SSL/TLS-PEM resources
+
+TODO: add a script/tool to regenerate the resources here.
```

Comment:
Perhaps remove this file if we're not going to introduce this script/tool yet?

Reply:
Fair enough, I'll create a ticket instead.

Reply:
Raised #29191